go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/openshift/api
+
+go 1.13
+
+require (
+	github.com/davecgh/go-spew v1.1.1
+	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
+	github.com/golang/protobuf v1.3.2
+	github.com/google/go-cmp v0.3.0
+	github.com/google/gofuzz v1.0.0
+	github.com/json-iterator/go v1.1.8
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
+	github.com/modern-go/reflect2 v1.0.1
+	github.com/openshift/library-go v0.0.0-20191205152556-73e1fb871a9b
```

Comment:
Given we copied the build machinery is it possible not to rely on library-go?

Reply:
Oh, nvmd, I see this is solved later on.

Reply:
This is the old version, the PR has been updated few days ago.